src/ReinforcementLearningEnvironments/src/environments/3rd_party/gym.jl

```diff
@@ -93,7 +93,7 @@ function RLBase.is_terminated(env::GymEnv{T}) where {T}
     end
 end
 
-function RLBase.is_truncated(env::GymEnv{T}) where {T}
+function is_truncated(env::GymEnv{T}) where {T}
```

Comment:
Why this change?

Reply:
Because it is not registered in RLBase (so far as I understood it, there must be a base version in interface.jl and CommonRLInterface.jl), it couldn't inherit from that, so I deleted the inheritance. It would be cleaner, if it also were in RLBase, but I did not want to interfere with that as I wasn't quite sure, whether this will impede some other functionalities...